web/app/globals.css

```
@@ -1,26 +1,48 @@
+  --bg: #ffffff;
+  --surface: #f7f8fa;
+  --fg: #111111;
+  --muted: #6b7280;
+  --border: #d9dee7;
+  --accent: #1677ff;
+  --accent-soft: color-mix(in oklch, var(--accent) 10%, transparent);
+  --accent-strong: color-mix(in oklch, var(--accent) 70%, black);
+  --font-body: Inter, system-ui, -apple-system, "Segoe UI", "Helvetica Neue", Arial, sans-serif;
+  --font-mono: ui-monospace, "SF Mono", "JetBrains Mono", Menlo, monospace;
+  --radius: 8px;
+  color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)